readthedocs/proxito/tests/test_redirects.py

```diff
@@ -83,7 +83,8 @@ def test_subproject_redirect(self):
         r = self.client.get('/', HTTP_HOST='subproject.dev.readthedocs.io')
         self.assertEqual(r.status_code, 302)
         self.assertEqual(
-            r['Location'], 'https://project.dev.readthedocs.io/projects/subproject/en/latest/',
+            r["Location"],
```

Comment:
Was this also wrong, or are we now doing 2 HTTP redirects here (this URL will 302 to `/projects/subproject/en/latest/`?

Reply:
This will do 2 redirects now, both will be cached.

Reply:
Should probably check for both in the test?

Reply:
we may already have one, I'm fine testing it here just to be sure.